tests/unit/request/util.js

```diff
@@ -24,16 +25,79 @@ define([
                     code: "B98765",
                     purchased: new Date(2017, 0, 1)
                 },
-                formData: formData,
                 durian: 100
             };
             util.deepCopy(object1, object2);
 			assert.strictEqual(object1.banana.weight, 52);
 			assert.strictEqual(object1.banana.price, 200);
 			assert.strictEqual(object1.banana.code, "B98765");
-			assert.strictEqual(object1.formData, formData);
 			assert.equal(object1.banana.purchased.getTime(), new Date(2017, 0, 1).getTime());
-		}
+		},
 
+        'deepCopy with FormData': function () {
+		    if (has('native-formdata')) {
```

Comment:
Indentation is inconsistent here

Reply:
> This PR has a lot of test fixes - are they all related to the `deepCopy` changes? If not, might it be better to split them into a different PR?

When I started working on this, there were a LOT of failures.  Some of them were from the previous changes to deepCopy.  Some were not.